client/src/pages/SymptomsCheck.js

```diff
@@ -481,10 +495,24 @@ const STEP_6_ANSWERS = [
   "I’m not sure",
 ];
 
+const STEP_6_STATE = {
+  answers: getAnswersMap(STEP_6_ANSWERS),
+  none: false,
+};
+
 const Step6 = (props) => {
-  const onSelectAnswer = (answer) => {
-    props.update("exposureLast14Days", answer);
-    props.nextStep();
+  const [state, updateState] = useState(STEP_6_STATE);
+  const { answers, none } = state;
+  const toggleAnswer = (answer) => {
+    const updatedAnswers = { ...answers, [answer]: !answers[answer] };
```

Comment:
For now it looks good but maybe we can use `useReducer` in the future to handle complex state